core/templates/dev/head/pages/admin/admin.html

```diff
@@ -70,6 +70,46 @@
 
     {% include 'pages/footer_js_libs.html' %}
 
+    <!-- Exploration player -->
```

Comment:
I think it is better to load the scripts only where they are needed. If it is possible to remove them from `base.html` and other main pages without breaking anything, we should do that. Once we finalize the loading of scripts [here](https://github.com/oppia/oppia/pull/6475/files#r278095172), all modularity PRs can follow the same pattern then.